Modules/Sources/Codegen/Sourcery/Copiable/WooFoundation-Copiable.sourcery.yaml

```diff
@@ -1,7 +1,17 @@
-project:
-    file: ../../../../../WooFoundation/WooFoundation.xcodeproj
-    target:
-        name: WooFoundation
-templates:
-    - Models+Copiable.swifttemplate
-output: ../../../../../WooFoundation/WooFoundation/Model/Copiable/
+configurations:
+    - sources:
+        include:
+            - ../../../../../Modules/Sources/WooFoundation
+      templates:
+          - Models+Copiable.swifttemplate
+      output: ../../../../../Modules/Sources/WooFoundation/Model/Copiable/
+      args:
+          moduleName: WooFoundation
+    - sources:
+        include:
+            - ../../../../../Modules/Sources/WooFoundationCore
+      templates:
+          - Models+Copiable.swifttemplate
+      output: ../../../../../Modules/Sources/WooFoundationCore/Model/Copiable/
+      args:
+          moduleName: WooFoundationCore
```

Comment:
To keep things neat and avoid compilation issues, I decided to have dedicated outputs for each of WooFoundation and WooFoundationCore.

At this time, WooFoundation does not have any copiable file, which result in an empty generated file. But I think it's better to generated an empty file than to risk expecting a file to be generated in the future and it not working because we removed the entry from here since at the time there was no need for it.